uint8 RAW_VIDEO=0
uint8 FARNBACK=1
uint8 MIX_OF_GAUSSIANS=2

uint8 mode
---
bool result
